docs/tutorial/file.rst

```diff
@@ -5,10 +5,10 @@ Files and channels
 
     :nbexport:`Download this page as a Jupyter notebook <self>`
 
-Opening a Bluelake HDF5 file is very simple::
+Open a Bluelake HDF5 file as follows::
 
     import lumicks.pylake as lk
-
+    
     file = lk.File("example.h5")
```

Comment:
Is the file still going to be called `example.h5`?

Reply:
in the future there should be a link to Zenodo on this page, I guess we can just keep it like this for now?

Reply:
It would likely still have a filename. But yes, it's fine to rename it later. 👍